server/src/main/java/org/elasticsearch/health/node/DiskHealthIndicatorService.java

```diff
@@ -237,7 +235,7 @@ String getSymptom() {
                     int unhealthyNodesCount = getUnhealthyNodeSize(masterNodes) + getUnhealthyNodeSize(otherNodes);
                     symptom += String.format(
                         Locale.ROOT,
-                        " Furthermore %d node%s with roles: [%s] %s out of disk or running low on disk space.",
+                        " %d node%s with roles: [%s] %s out of disk or running low on disk space.",
                         unhealthyNodesCount,
                         unhealthyNodesCount == 1 ? "" : "s",
```

Comment:
Maybe we can use `regularNoun("node", dataNodes.size())` here too. I see I missed it.